pkg/grpc/exec/exec.go

```diff
@@ -288,6 +288,15 @@ func GetProcessExit(event *MsgExitEventUnix) *tetragon.ProcessExit {
 	code := event.Info.Code >> 8
 	signal := readerexec.Signal(event.Info.Code & 0xFF)
 
+	// Ensure that we get PID == TID
+	if event.ProcessKey.Pid != event.Info.Tid {
+		logger.GetLogger().WithFields(logrus.Fields{
```

Comment:
I'm tempted to say just remove these blocks. So it should never happen AFAICT and if it did happen in some rare instance I doubt debugging is enabled :/

Reply:
I would agree, but we have the same bit in exec path and it helped uncover some paths that were missing between in some versions (variants) of tetragon... beside the bpf and userspace bits ;-)  , and for exec path it was not a debug message we made it an info or warn then we turned it to debug, after fixing all those...

For debugging enabled, we have a way now to turn it on during runtime, if it doesn't hurt lets please keep it as it could help point to the problem quickly in such cases...

Reply:
hmm seems it was still warn() not debug, and I moved it a bit early but kept it warn() for now, see last patch. In my mind it was: run with warn() then turn into debug() after some time and I thought we already did...  anyway all warnings have been fixed but lets see...